File: javascript/calculadora-medias.js
const prompt = require ("prompt-sync")();

const aluno = "NomeAluno"
let pontosSomados = 0;
let notasLancadas = 0; 
let mediaPontos = 0;

while (notasLancadas <5) {
const nota = Number(prompt(`${notasLancadas + 1}a nota do ${aluno}: `));
console.log(`A nota iserida foi ${nota}`); 
pontosSomados = pontosSomados + nota;
notasLancadas = notasLancadas + 1;
mediaPontos = pontosSomados / notasLancadas
console.log(`Média até ${notasLancadas}a nota: ${mediaPontos}`);
}

console.log(`Média final do ${aluno}: ${mediaPontos}`)

if (mediaPontos >=6){ 
    console.log("Aluno aprovado");
} else { 
    console.log("Aluno reprovado");
}

File: python/calculadora-medias.py
aluno = "Nome do aluno"
pontos_somados = 0
notas_lancadas = 0
media_pontos = 0

while notas_lancadas < 5:
    nota = float(input(f"{notas_lancadas + 1}ª nota do {aluno}: "))
    print(f"A nota inserida foi {nota}")
    pontos_somados += nota
    notas_lancadas += 1
    media_pontos = pontos_somados / notas_lancadas
    print(f"Média até {notas_lancadas}ª nota: {media_pontos}")

print(f"Média final do {aluno}: {media_pontos}")

if media_pontos >= 6:
    print("Aluno aprovado")
else:
    print("Aluno reprovado")
